packages/cli/package.json

```diff
@@ -248,6 +248,32 @@
                 "description": "The port that the frontend should be bound to. Only used for local development."
               }
             }
+          },
+          "https": {
+            "type": "object",
+            "description": "Running the frontend app with https",
```

Comment:
Maybe we should clarify like above that it's only used for local development in these descriptions too, and that they're passed to wepback only.